internal/api/api_test.go

```diff
@@ -202,7 +225,11 @@ func (fake *fakeGCS) RoundTrip(req *http.Request) (*http.Response, error) {
 
 	content, err := os.Open(filename)
 	if err != nil {
-		fake.Fatalf("Failed to open test data: %v", err)
+		return &http.Response{
```

Comment:
Use http.Error for this

Reply:
http.Error takes a http.ResponseWriter as argument which I don't have. can I obtain it from the request?

Reply:
I was imagining:
response := httptest.NewRecorder()
http.Error(response, ...)
return response.Result(), nil

Reply:
thanks